### PR TITLE
Renamed examples per guidelines and added TDMS example

### PR DESCRIPTION
**Pull request by fredvisser**
This PR substantially improves the Call SystemLink Data Services example project by deleting the 'mobile' examples, renaming all examples to be more descriptive, adding a basic TDM Reader API example, updating the remaining Basic examples to match the TDM style, and by refactoring the Full example.

This project was built in NXG 4.0.

### CODE REVIEW

**Comment by funkmaster2** [comment]
I added Sid to this because he's going to do the integration.

**Comment by rajsite** [comment]
@fredvisser Can you revert the build folder changes in the PR? It adds a lot of noise during review. We will add new builds on release.

Also we should target the changes to a branch since Marmot isn't released yet. I'll make a 4.0 branch for you to target.

**Comment by rajsite** [comment]
I created the nxg4 branch here: https://github.com/ni/webvi-examples/tree/nxg4

**Comment by rajsite** [comment]
- `Update Remaining Queue Indicator.gvi` has a conditional disable in it, can that be removed?
- Running index.gviweb without entering credentials or pressing Connect immediately causes an error dialog in the editor because of the queue size initialization. Maybe instead the queue can be initialized in the connect-success case?
- It looks like there is an unused `Queue FGV.gvi`, can we remove that?
- It looks like some images were deleted from the readme, can you delete the image files that are unused from readme_files as well?

File: CallSystemLinkDataServices/Readme.md [hunk 1]
```diff
@@ -1,236 +1,228 @@
...
-
-If storing credentials in a file is not an option and you can host your web application in the cloud, you may choose to host your web application on SystemLink Cloud. This requires you to have an active SSP for the LabVIEW NXG Web Module and an ni.com account. Doing so allows an authentication cookie in your browser to be automatically sent with calls to SystemLink Cloud. This eliminates the need to enter an API key or server URL either in the web application.
+# Call SystemLink Data Services
+
+[![Call SystemLink Data Services](https://img.shields.io/badge/Details-Demo_Link-green.svg)](https://ni.github.io/webvi-examples/CallSystemLinkDataServices/Builds/WebApp_Web%20Server/)
+[![Call SystemLink Data Services README Link](https://img.shields.io/badge/Details-README_Link-orange.svg)](https://github.com/ni/webvi-examples/tree/master/CallSystemLinkDataServices)
```
Comment (rajsite) [inline CallSystemLinkDataServices/Readme.md]: The url should stay as empty. ie
```
[![Call SystemLink Data Services README Link](https://img.shields.io/badge/Details-README_Link-orange.svg)]()
```

File: CallSystemLinkDataServices/Readme.md [hunk 2]
```diff
@@ -1,236 +1,228 @@
...
+### Hosting a Package File (.nipkg) on the NI Web Server
+
+1. Open `CallSystemLinkDataServices.lvproject`
+2. Open `Full Data Services App.lvdist`.
+3. Click the **Build distribution**.
+   > **Note**: The button looks like a hammer.
+
+4. Click **Locate item in Windows Explorer** to find the build output.
+5. Double-click the package (`.nipkg`) and follow the on-screen instructions.
+6. Open a web browser and navigate to `http://localhost:9090/CallSystemLinkdataservices/index.html`.
```
Comment (rajsite) [inline CallSystemLinkDataServices/Readme.md]: shouldn't need the `index.html` in the url

File: CallSystemLinkDataServices/Readme.md [hunk 3]
```diff
@@ -1,236 +1,228 @@
...
+
+#### Hosting on the LabVIEW 2009-2018 Web Server
+
+1. Go to the LabVIEW Web Server directory using the following path: `C:\Program Files (x86)\National Instruments\Shared\NI WebServer\www`.
+2. Copy the entire `Full Data Services App_Web Server` directory into the `www` directory of the LabVIEW web server.
+3. Open a web browser and navigate to `http://localhost:8080/Full%20Data%20Services%20App_Web%20Server/index.html`.
```
Comment (rajsite) [inline CallSystemLinkDataServices/Readme.md]: shouldn't need the index.html

File: CallSystemLinkDataServices/Readme.md [hunk 4]
```diff
@@ -1,236 +1,228 @@
...
+
+#### Hosting on the NI Web Server
+
+1. Go to the NI Web Server using the following path: `C:\Program Files\National Instruments\Shared\Web Server\htdocs`.
+2. Copy the `Full Data Services App_Web Server` directory into the `htdocs` directory of the NI Web Server.
+3. Open a web browser and navigate to `http://localhost:9090/Full%20Data%20Services%20App_Web%20Server/index.html`.
```
Comment (rajsite) [inline CallSystemLinkDataServices/Readme.md]: One more index.html not really needed

**Comment by rajsite** [comment]
I think it might be useful to have the README.md describe how to find the NXG Desktop and CG examples.

File: CallSystemLinkDataServices/Readme.md (hunk 4 above)
Comment (fredvisser) [reply CallSystemLinkDataServices/Readme.md]: Removed

File: CallSystemLinkDataServices/Readme.md (hunk 3 above)
Comment (fredvisser) [reply CallSystemLinkDataServices/Readme.md]: Removed

File: CallSystemLinkDataServices/Readme.md (hunk 2 above)
Comment (fredvisser) [reply CallSystemLinkDataServices/Readme.md]: Removed

File: CallSystemLinkDataServices/Readme.md (hunk 1 above)
Comment (fredvisser) [reply CallSystemLinkDataServices/Readme.md]: Done

**Comment by fredvisser** [comment]
Thanks for the great feedback - I've incorporated all of it. Let me know if anything else comes up.

**Comment by rajsite** [comment]
@fredvisser sorry, one more minor thing I missed, can you delete the old lvprojectcache?

File: .spelling [hunk 5]
```diff
@@ -93,3 +93,6 @@ SystemLink
 webvi-examples
 USGS
 DataQueue
+LabVIEWExamples.lvproj
```
Comment (rajsite) [inline .spelling]: We have been marking filenames / variable names, etc as code pieces and the spellcheck will avoid spelling errors on code pieces. ie wrap them in backticks like
```
`LabVIEWExamples.lvproj`
```

You can see that convention used for other paths and filenames in the READMEs

Comment (fredvisser) [reply .spelling]: Ah, that's the trick - done.

**Comment by fredvisser** [comment]
I had missed the rest of the feedback yesterday, but it should be good now. 

For the LabVIEWExamples folder, I expanded the discussion of it (slightly) in the readme, and it is mentioned in the instructions to the Read Tags examples, as that's where it's needed.

**Comment by rajsite** [comment]
@fredvisser was able to generate a new lvprojectcache (and caught a couple of minor things).
Thanks for all the work, looks ready to go!

**Pull request merged**